source/core/slang-shared-library.cpp

```diff
@@ -22,6 +22,7 @@ static const Guid IID_ISlangSharedLibraryLoader = SLANG_UUID_ISlangSharedLibrary
     "d3dcompiler_47",           // SharedLibraryType::Fxc
     "slang-glslang",            // SharedLibraryType::Glslang  
     "dxil",                     // SharedLibraryType::Dxil
+    "nvrtc64_102_0",            // SharedLibraryType::NVRTC
```

Comment:
The embedded version number there indicates that we will eventually want to expand this to have multiple names to search for per compiler.